workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/computed_diagnostics.py

```diff
@@ -52,6 +52,21 @@ def url_to_folder(url):
             {str(k): url_to_folder(url) for k, url in enumerate(urls)}
         )
 
+    @staticmethod
+    def from_json(url: str) -> "ComputedDiagnosticsList":
+        """Open labeled computed diagnostics at urls specified in given JSON."""
+
+        def url_to_folder(url):
+            fs, _, path = fsspec.get_fs_token_paths(url)
```

Comment:
nit: Since this code depends on `vcm` anyway, I would lean towards using the `vcm.cloud.get_fs` function.

Reply:
I am going to leave it as, since we due use the `path` output of the function here. I copied this from the existing `from_urls` method.